src/utils.js

```diff
@@ -1,5 +1,6 @@
 "use strict";
 
+const axios = require("axios");
 const p = require("path");
```

Comment:
use `path` for that, abbreviating to one letter variables reduces readability

Reply:
k, blame points to you though 😉